### DISCUSSION TITLE
Top rated games page (game discoverability)

### DISCUSSION THREAD

**Discussion by Krylan** [RetroAchievements/RAWeb#959] [Ideas]
I stumbled with the idea of making a game listing, which would be sorted by star ratings (those where users on game page can vote). It could help users in discovering interesting games to play, which are acclaimed by RA community. 
There could be also a module for "Random game(s)" or others. In future, we could even go step further, ex. display highest rated developers (by aggregating average score of their games).
_EDIT: Just to be clear, by "developers" I mean game creators, not achievement set creators_

System should take into account number of votes, so it would exclude items, which doesn't have enough of them (it would avoid the situation, where  game with one vote of 5/5 would get it right on the top of list).

This could be a separate page or module/section on other, or by using an existing one (like all games with column for rating).

**Comment by StingX2** [comment]
The top rated games is already a confusion for users if they are scoring the game or the set, or in the cases of the petty users on site how can they review bomb people they don't like. So yeah games inherently are innocent but the later part is what will cause issue.

It's bad enough the developer leaderboard for achievements made or yielded unlocks is seen as competitive. Showcasing devs like this is only going to lead to more competition or outright turmoil. We already lose devs each year that can't handle tickets, devs targetted by users that don't like them, or devs burnt out by the administration. I don't think we need to add a statistic that can be controlled where people that don't like a certain dev can abuse it. That dev may not care or it may cost the site another developer.

**Reply by Tsearo** [reply]
That's a good point about dev ratings and review bombing that I didn't think about. I can definitely see that becoming a discouragement for some developers depending on the severity of it 🙁

**Reply by Krylan** [reply]
Ouch, I think there was some misunderstanding because of disambiguous wording. By "developer" in my post, I meant game creator, not achievement set creator. So it would be like ranking of game companies to see, if Sega has higher rated games overall than Konami etc.
When posting, I didn't though that it could be seen as saying about set devs, sorry 😅 

About confusion with voting on game/set, I understand the point – at first I also wasn't sure, what exactly there is rated (even if it says surely "Game Rating"). This part should be more clear for users, and we could also use those results to enchance search results in search engines (so it would show game rating on game result in Google, for example). About review bombing – you can never completely avoid that. Despite there is a ranking or not, some users would spitefully vote just to change the result to the extremum. The most important is to develop a culture of community to give more possibilities to use the data.

**Reply by StingX2** [reply]
Oh GAME creator makes much sense, I like that.

**Comment by Tsearo** [comment]
I like this idea, being able to sort on a new metric could be a cool feature for users to find games to play. On a similar note, one thing I've always felt we should do is add a set rating as well which imo would be more useful information for our users since there are already plenty of game ratings out there people can find. 
I still think there is some value in having game rating though as it is another metric that could be searched on (possibly we could pull this data from another existing source though for that?). Set Rating would be a better metric to aggregate score on for highest rated developers though imo otherwise it will just be showing devs that make sets for popular titles. The main caveat/concern with this though is that some users would likely rate harder sets lower and easy sets higher which could skew results and make it less useful.
Which would then lead to a full list of ratings such as set design, set difficulty, set length, set enjoyableness, etc.
Hope I didn't derail too much with all of that 😄 Back to the current topic of adding it for current Game Rating. I think it would likely help motivate more users to start rating games if we see some benefit from being able to use the data which is a good thing. I also like the idea of being able to sort the existing games (all or by console) by rating personally although a new page that could display the top 5 or so rated games for each console as highlights  with a  link to the respective consoles game listing sounds like it could compliment the idea further.

**Reply by Jamiras** [reply]
> one thing I've always felt we should do is add a set rating

We used to have a set rating and it was removed because people were abusing it.

https://discord.com/channels/310192285306454017/310195377993416714/524411337858875392

> It was removed because it appeared some users were manipulating votes targeting some devs without even touching the set. And also there was not enough voting in general to give a strong rating of the set.

See #663 for reviving it. The idea is that there needs to be a minimum number of votes before it become active (as suggested in the first post here), and only people with a certain completion percentage are allowed to vote.

**Reply by Tsearo** [reply]
Oh nice, adding a set completion threshold to being able to vote on set rating sounds like a decent solution to that but would be tricky on deciding what a good threshold would be 🤔
